calico/datamodel_v1.py

```diff
@@ -85,6 +86,10 @@
 ENDPOINT_STATUS_DOWN = "down"
 ENDPOINT_STATUS_ERROR = "error"
 
+# Information intended for use by the DHCP agent.
+DHCP_DIR = ROOT_DIR + "/dhcp" + DHCP_VERSION
+SUBNET_DIR= DHCP_DIR + "/subnet"
```

Comment:
Well I think so, from a DHCP perspective, for the same reason as commented at lines 121-128 of https://review.openstack.org/#/c/241310/17/networking_calico/agent/dhcp_agent.py.